resources/Materials/TestSuite/stdlib/texture/texcoord.mtlx

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
```

Comment:
Actually, color nodes do not suffer from the same problem because they are always typed as `COLOR4` in the generated GLSL/MSL code, regardless of their actual width (so a graph which only has a `geomcolor_color3` will still emit an input variable of type `color4` in the variable block).

I think it would be best to first merge this PR and create another one if we want to support codegen with a different type depending on the width of the color nodes in the graph.

Reply:
See e.g. `MaterialXGenGlsl/Nodes/GeomColorNodeMsl.cpp`.